src/jdk.jpackage/share/classes/jdk/jpackage/internal/resources/MainResources_de.properties

```diff
@@ -75,8 +75,9 @@ error.blocked.option=jlink-Option [{0}] ist in --jlink-options nicht zul\u00E4ss
 error.no.name=Name nicht mit --name angegeben. Es kann auch kein Name aus app-image abgeleitet werden
 
 warning.no.jdk.modules.found=Warnung: Keine JDK-Module gefunden
-warning.foreign-app-image=Warnung: app-image-Verzeichnis ({0}) wurde von jpackage nicht generiert.
-warning.invalid-app-image=Warnung: .jpackage.xml kann in app-image-Verzeichnis ({0}) nicht geparst werden
+
+error.foreign-app-image=Error: app-image dir ({0}) not generated by jpackage. Missing .jpackage.xml.
```

Comment:
This error message will be misleading in case app image was generated by jpackage and adjusted after.
I'd put it as "Missing .jpackage.xml file in app-image dir ({0})."

Reply:
Done.